utils/staking-miner/src/main.rs

```diff
@@ -351,7 +355,10 @@ struct SharedConfig {
 	#[clap(long, short, default_value = DEFAULT_URI, env = "URI")]
 	uri: String,
 
-	/// The seed of a funded account in hex.
+	/// The path to a file containing the seed of the account. If the file is not found, the seed is
```

Comment:
> If the file is not found, the seed is used as-is.

Can you clarify the documentation that the actual seed can be provided as `file path` or `the actual seed`?
Maybe it's just me

additionally would be good to rename `seed` -> `seed_or_path` to would help someone like me that sometimes uses the `CLI --help stuff`

Reply:
What I say is exactly the behavior: if it is an existent file, it is read and its content is used. If non-existent, then we treat it as seed data.